client/python/gradio_client/utils.py

```diff
@@ -856,9 +856,9 @@ def _json_schema_to_python_type(schema: Any, defs) -> str:
     elif type_ == "null":
         return "None"
     elif type_ == "const":
-        return f"Litetal[{schema['const']}]"
+        return f"Literal[{schema['const']}]"
     elif type_ == "enum":
-        return f"Literal[{', '.join([str(v) for v in schema['enum']])}]"
+        return "Literal[" + ", ".join(["'" + str(v) + "'" for v in schema['enum']]) + "]"
```

Comment:
Unrelated fix that just adds quotation marks around literals. So instead of the type hint in the API docs being: `Literal[a]`, it will be correctly displayed as `Literal["a"]`